mockito/mocking.py

```diff
@@ -77,7 +77,7 @@ def clear_invocations(self):
     def get_original_method(self, method_name):
         """
         Looks up the original method on the `spec` object and returns it
-        together with an indicatin of whether the the method is found
+        together with an indication of whether the the method is found
```

Comment:
yeah, it was this plus the double "the". 😄  Minor things.

Reply:
d'oh!  fixed.

Reply:
a clear case of "author-goggles"  ; )